src/models/index.ts

```diff
@@ -0,0 +1,23 @@
+export interface Schedule {
+	classes: ClassMap
+	semesters: SemesterMap
+}
+
+export interface ClassMap {
+	[id: string]: Class
+}
+
+export interface Class {
+	id: string
+	title: string
+}
+
+export interface SemesterMap {
+	[id: string]: Semester
+}
+
+export interface Semester {
+	id: string
+	title: string
+	classIds: string[]
+}
```

Comment:
I also added these as part of the backend in what will be backend/src/types.ts, unfortunately in a different branch though